vxquery-core/src/main/resources/org/apache/vxquery/xmlquery/exceptions/XMLQueryErrorMessages.properties

```diff
@@ -139,4 +139,5 @@ ERR_FORX0001_DESCRIPTION=Invalid regular expression. flags
 ERR_FORX0002_DESCRIPTION=Invalid regular expression.
 ERR_FORX0003_DESCRIPTION=Regular expression matches zero-length string.
 ERR_FORX0004_DESCRIPTION=Invalid replacement string.
-ERR_FOTY0012_DESCRIPTION=Argument node does not have a typed value.
\ No newline at end of file
+ERR_FOTY0012_DESCRIPTION=Argument node does not have a typed value.
```

Comment:
Please add all the JSONiq error messages: http://jsoniq.org/docs/JSONiqExtensionToXQuery/html-single/index.html#section-error-codes